cmd/root.go

```diff
@@ -233,6 +236,14 @@ func deleteLogs(logger *zap.Logger) {
 		return
 	}
 }
+func checkForVersionFlag(args []string) bool {
```

Comment:
I think the code is not yet updated with the main branch. Please have a look

Reply:
Updated